tests/baselines/reference/user/acorn.log

```diff
@@ -275,6 +275,9 @@ node_modules/acorn/dist/acorn.js(5299,5): error TS2339: Property 'nextToken' doe
 node_modules/acorn/dist/acorn.js(5300,12): error TS2339: Property 'parseExpression' does not exist on type 'Parser'.
 node_modules/acorn/dist/acorn.js(5307,10): error TS2554: Expected 3 arguments, but got 2.
 node_modules/acorn/dist/acorn_loose.es.js(45,12): error TS2554: Expected 3 arguments, but got 2.
+node_modules/acorn/dist/acorn_loose.es.js(64,7): error TS2532: Object is possibly 'undefined'.
```

Comment:
This is from Anders' new control flow checking of this-property assignments in the constructor -- acorn's Node only initialises about half of its properties under certain config settings.